cylc/flow/cfgspec/workflow.py

```diff
@@ -68,19 +101,28 @@
 ) as SPEC:
 
     with Conf('meta', desc='''
-        Section for metadata items for this workflow. Several items (title,
-        description, URL) are pre-defined and are used by Cylc. Others can be
-        user-defined and passed to workflow event handlers to be interpreted
-        according to your needs. For example, the value of "workflow-priority"
-        could determine how an event handler responds to failure events.
+        Section for metadata items for this workflow. Cylc defines and uses
+        some terms (title, description, URL). Users can define more terms,
+        and use these in event handlers.
+
+        .. note::
+
+           A user could define "workflow-priority". An event handler
+           would then respond to failure events in a way set by
+           "workflow-priority".
     '''):
         Conf('description', VDR.V_STRING, '', desc='''
             A multi-line description of the workflow. It can be retrieved at
             run time with the ``cylc show`` command.
+
+            .. TODO Does this still work?
```

Comment:
Does Cylc Show work any more?

Reply:
Yes.

Reply:
(Use for seeing detailed state of task prerequisites and outputs)